src/markdown/features/routes.md

```diff
@@ -0,0 +1,77 @@
+# Defining Routes
+
+Routes are defined in the `lenra.yml` file under the `lenraRoutes` property. The `lenraRoutes` property is an array of objects, where each object represents a route. Each route object has two properties:
```

Comment:
It's not defined in the lenra.yml file but in the app manifest.

Why do we use an array instead of a map ? 
```json
"/path": {
  "type": "view",
  "name": "myView"
}
```

Reply:
It was a personnal preference but I can change it to match this format instead.

Reply:
The most readable one : :heart:

Reply:
The map won't work, the array is mandatory..

The order of the routes are super important and since the map cannot ensure the order, we MUST use an array.

Reply:
With an example : 
/foo/bar
/foo/:name

Those are two correct routes, and the order is important. If we switch them : 
/foo/:name
/foo/bar

the route /foo/bar wil never match.

Reply:
After some discussion with @Nesqwik we will keep the array format.